src/index.ts

```diff
@@ -14,6 +14,11 @@ interface LogifyPropsType<ErrorGenericType> {
     error?: string
     fatal?: string
   }
+  storage?: {
+    key: string
+    get: (key: string) => string | null
+    set: (key: string, value: string) => any
+  }
```

Comment:
for the sake of consistency, i'd prefer us using the same properties as [here](https://github.com/colorfy-software/react-native-gigya-sdk/blob/main/src/types.ts#L7-L10).

Reply:
could you use _exactly_ the same value for `getItem()` and `setItem()` pls? the reason why i'm asking this specifically is because some storage solution do not expose synchronous getters/setters.